src/components/WalletDropdown/AuthenticatedHeader.tsx

```diff
@@ -81,6 +81,10 @@ const HeaderWrapper = styled.div`
   justify-content: space-between;
 `
 
+const AuthenticatedHeaderWrapper = styled.div`
```

Comment:
this component and some other where in in a parent wrapper with 16px padding.  In order to get the hover state right, I had to move that padding to the children components.  This is why this has padding now